storybook/cypress/e2e/ozon-content.cy.ts

```diff
@@ -40,7 +40,7 @@ describe("Ozon Content", () => {
     button("N7").should("have.attr", "aria-expanded", "true");
     tooltip("N7").should("be.visible");
 
-    // cy.percySnapshot(`${Cypress.currentTest.title} -- visible notes`);
+    cy.get("dso-ozon-content.hydrated").wait(250).matchImageSnapshot(`${Cypress.currentTest.title} -- visible notes`);
```

Comment:
Die 250ms is zeker de duur van de animatie van de tooltip? Wil je dat aanvullen met een commentaartje?

Reply:
Ik denk dat ik die niet op de duur van de animatie van de tooltip heb gebaseerd, want ik weet niet waar die duur staat ingesteld.

Reply:
![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/b4aa198f-0290-4e56-b9e5-f946d6f5c0ec)

Toch een verschil met die 150ms. Dan maar 250ms, maar laten we wel een commentaartje plaatsen dat die wait nodig is voor de animatie van Tooltip.

Reply:
Gedaan

Reply:
Toppertje